src/rkopenmdao/integration_control.py

```diff
@@ -72,6 +73,16 @@ def remaining_time(self):
         else:
             raise TypeError("Termination criteria must be end_time.")
 
+    def iteration_control(self):
```

Comment:
I find the name `iteration_control` a bit undescriptive. Something like `iteration_criterion_not_reached` or `iteration_is_unfinished` (or something similar) tells more about what the function does.